components/builder-api/src/http/handlers.rs

```diff
@@ -62,17 +70,26 @@ pub fn job_create(req: &mut Request) -> IronResult<Response> {
             _ => return Ok(Response::with(status::UnprocessableEntity)),
         }
     }
-    let session = req.extensions.get::<Authenticated>().unwrap();
+    let session = req.extensions.get::<Authenticated>().unwrap().clone();
```

Comment:
I don't think you need to clone here, you're allocating a new string on line 89 by calling `session.get_id().to_string()` even though you're only using a ref of it in JobCreate